mods/d2k/weapons/other.yaml

```diff
@@ -107,35 +107,18 @@ Demolish:
 		ImpactActors: false
 
 Atomic:
-	Warhead@1Dam: SpreadDamage
-		Spread: 1c0
-		Falloff: 200, 100, 50, 25, 12, 0
-		Damage: 27000 ##225 in vanilla but of course is a cluster bomb instead, so damage spread out
-		Versus:
-			none: 90
-			wall: 50
-			building: 75
-			wood: 60
-			light: 60
-			heavy: 60
-			invulnerable: 0
-			cy: 25
-			harvester: 60
-		DamageTypes: Prone50Percent, TriggerProne, SoundDeath
+	Warhead@Cluster: FireCluster
+		Weapon: NukeDebris
+		RandomClusterCount: 10
+		Dimensions: 3,3
+		Footprint: xxx xxx xxx
```

Comment:
Looks like you did this wrong way around. TS Cluster missile is the one that is meant to fire in a 3x3 area and D2k one fires randomly on a bigger area.

Reply:
I looked at the original DH explosion, and most of the time most of the cluster would land very close to the target center.
Also, keep in mind that the weapon projectiles' inaccuracy effectively increases the max possible distance from the targeted cell.

Reply:
Checked TS too, but that only confirmed that my settings are pretty close to the original, they occasionally *did* stry quite far from the target location.

I did tweak the inaccuracy offsets (and damage) a bit, though.